internal/runtime/funcs/int_subtracter.go

```diff
@@ -0,0 +1,47 @@
+package funcs
+
+import (
+	"context"
+	"github.com/nevalang/neva/internal/runtime"
+)
+
+type intSubtracter struct{}
+
+func (intSubtracter) Create(io runtime.FuncIO, _ runtime.Msg) (func(ctx context.Context), error) {
+	streamIn, err := io.In.Port("stream")
+	if err != nil {
+		return nil, err
+	}
+
+	resOut, err := io.Out.Port("res")
+	if err != nil {
+		return nil, err
+	}
+
+	return func(ctx context.Context) {
+		flag := false
+		var res int64
+		for {
+			select {
+			case <-ctx.Done():
+				return
+			case streamItem := <-streamIn:
+				if streamItem == nil {
+					select {
+					case <-ctx.Done():
+						return
+					case resOut <- runtime.NewIntMsg(res):
+						res = 0
+						continue
+					}
+				}
+				if !flag {
```

Comment:
nit

How about using `var res *int64` and have 1 variable instead of 2? Only if that will make code simpler

Reply:
This will complicate the code for no benefit IMO